### ISSUE TITLE
Composer detected issues in your platform

### ISSUE THREAD

**Issue by KickSchutte67** [weprovide/valet-plus#610]
- [x] I've checked the issue queue and could not find anything similar to my bug.
- [x] I'm on the latest version of valet-plus (`valet --version`): `2.4.0-beta4`
- [x] I've run `valet fix` and `valet install` after updating and before submitting my issue/feature.

**What is the problem?**
I reinstalled my Valet+ to the 2.4.0-beta4 version to support PHP 8.1. After installation, my composer tells me `Fatal error: Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.1.0". You are running 8.0.23. in /Users/kickschutte/sites/huts-project/vendor/composer/platform_check.php on line 24` even though I am on 8.1.10 when I am doing `php -v`. 

I've relinked and ran the `valet use 8.1` several times, but I keep getting this error. Valet fix reverts me back to version 7.4. I also updated my composer and installed all composer packages. I also ran npm commands e.g. `npm install`.

<img width="378" alt="image" src="https://user-images.githubusercontent.com/31417270/188333998-6c390940-c191-4a19-9c01-09df082b1832.png">

<img width="349" alt="image" src="https://user-images.githubusercontent.com/31417270/188334009-a127f3f7-f40b-4ee6-beb4-b9a6102e6c38.png">

**Comment by KickSchutte67** [comment]
Update;

I was using the normal "php" version instead of "valet-php@8.1" from henkrehorst. I tried reinstalling valet but it gives me the: `Unable to determine linked PHP` error.

**Comment by sdouma** [comment]
@KickSchutte67  how did you fix it?

**Comment by JoshuaLuckers** [comment]
@KickSchutte67 did you manage to solve this issue?

**Comment by rsisco** [comment]
For anyone else that comes upon this issue, I found that in the `2.4.0-beta` versions the PHP path check in `valet-plus/cli/Valet/PhpFpm.php` only checks for `/php` or `php@`. A simple fix is just to patch or change the conditional at line 421 to:
```
if (strpos($resolvedPath, '/php@' . $version . '/') !== false ||
    strpos($resolvedPath, '/php/' . $version . '') !== false ||
    strpos($resolvedPath, '/valet-php@' . $version . '/') !== false
) {
     return $version;
}
```
I don't have the energy at the moment to dig in further to why it was changed, but I'm guessing it is related to the use of `'shivammathur/php/'` as the formula prefix at the beginning of the script.

EDIT:
Yes, according to `INSTALL.MD`, we should be moving away from the `valet-php@` versions from `henkrehorst` and to the `php@` versions from `shivammathur`.
